docs/requirements-doc.txt

```diff
@@ -0,0 +1,3 @@
+doc8==0.8.1
+Sphinx==3.3.1
+sphinx-rtd-theme==0.5.0
```

Comment:
I do not think we need a separate requirements-doc.txt file. We should probably add these here: https://github.com/nexB/skeleton/blob/ef210cd813de2961fe8ca4a5ec7f14532ea6e9f8/pyproject.toml#L2

Reply:
You're right. There seems to be a way to include it [like this](https://docs.readthedocs.io/en/latest/config-file/v1.html#python-extra-requirements). Doing this.